src/lumigoLogger.js

```diff
@@ -0,0 +1,27 @@
+import { validateTag } from './globals';
+
+module.exports = {
+  info: (message, type = 'ProgrammaticInfo', extra = {}) => {
```

Comment:
Can we add a doc about the possible arguments here?
what are the limitations on `extra`?